templates/chart-maps.html

```diff
@@ -1,102 +1,87 @@
+	['CsMilitia',    {{.UserStatsForGame.Stats.TotalWinsMapCsMilitia}},    {{.UserStatsForGame.Stats.TotalRoundsMapCsMilitia}}    - {{.UserStatsForGame.Stats.TotalWinsMapCsMilitia}}],
+	['CsOffice',     {{.UserStatsForGame.Stats.TotalWinsMapCsOffice}},     {{.UserStatsForGame.Stats.TotalRoundsMapCsOffice}}     - {{.UserStatsForGame.Stats.TotalWinsMapCsOffice}}],
+	['DeAztec',      {{.UserStatsForGame.Stats.TotalWinsMapDeAztec}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeAztec}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeAztec}}],
+	['DeBank',       {{.UserStatsForGame.Stats.TotalWinsMapDeBank}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeBank}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeBank}}],
+	['DeCbble',      {{.UserStatsForGame.Stats.TotalWinsMapDeCbble}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeCbble}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeCbble}}],
+	['DeDust',       {{.UserStatsForGame.Stats.TotalWinsMapDeDust}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeDust}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeDust}}],
+	['DeDust2',      {{.UserStatsForGame.Stats.TotalWinsMapDeDust2}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeDust2}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeDust2}}],
+	['DeInferno',    {{.UserStatsForGame.Stats.TotalWinsMapDeInferno}},    {{.UserStatsForGame.Stats.TotalRoundsMapDeInferno}}    - {{.UserStatsForGame.Stats.TotalWinsMapDeInferno}}],
+	['DeLake',       {{.UserStatsForGame.Stats.TotalWinsMapDeLake}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeLake}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeLake}}],
+	['DeNuke',       {{.UserStatsForGame.Stats.TotalWinsMapDeNuke}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeNuke}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeNuke}}],
+	['DeSafehouse',  {{.UserStatsForGame.Stats.TotalWinsMapDeSafehouse}},  {{.UserStatsForGame.Stats.TotalRoundsMapDeSafehouse}}  - {{.UserStatsForGame.Stats.TotalWinsMapDeSafehouse}}],
+	['DeShorttrain', {{.UserStatsForGame.Stats.TotalWinsMapDeShorttrain}}, {{.UserStatsForGame.Stats.TotalRoundsMapDeShorttrain}} - {{.UserStatsForGame.Stats.TotalWinsMapDeShorttrain}}],
+	['DeStmarc',     {{.UserStatsForGame.Stats.TotalWinsMapDeStmarc}},     {{.UserStatsForGame.Stats.TotalRoundsMapDeStmarc}}     - {{.UserStatsForGame.Stats.TotalWinsMapDeStmarc}}],
+	['DeSugarcane',  {{.UserStatsForGame.Stats.TotalWinsMapDeSugarcane}},  {{.UserStatsForGame.Stats.TotalRoundsMapDeSugarcane}}  - {{.UserStatsForGame.Stats.TotalWinsMapDeSugarcane}}],
+	['DeTrain',      {{.UserStatsForGame.Stats.TotalWinsMapDeTrain}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeTrain}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeTrain}}],
+	['DeVertigo',    {{.UserStatsForGame.Stats.TotalWinsMapDeVertigo}},    {{.UserStatsForGame.Stats.TotalRoundsMapDeVertigo}}    - {{.UserStatsForGame.Stats.TotalWinsMapDeVertigo}}]
```

Comment:
```suggestion
	['Baggage',    {{.UserStatsForGame.Stats.TotalWinsMapArBaggage}},    {{.UserStatsForGame.Stats.TotalRoundsMapArBaggage}}    - {{.UserStatsForGame.Stats.TotalWinsMapArBaggage}}],
	['Monastery',  {{.UserStatsForGame.Stats.TotalWinsMapArMonastery}},  {{.UserStatsForGame.Stats.TotalRoundsMapArMonastery}}  - {{.UserStatsForGame.Stats.TotalWinsMapArMonastery}}],
	['Shoots',     {{.UserStatsForGame.Stats.TotalWinsMapArShoots}},     {{.UserStatsForGame.Stats.TotalRoundsMapArShoots}}     - {{.UserStatsForGame.Stats.TotalWinsMapArShoots}}],
	['Assault',    {{.UserStatsForGame.Stats.TotalWinsMapCsAssault}},    {{.UserStatsForGame.Stats.TotalRoundsMapCsAssault}}    - {{.UserStatsForGame.Stats.TotalWinsMapCsAssault}}],
	['Italy',      {{.UserStatsForGame.Stats.TotalWinsMapCsItaly}},      {{.UserStatsForGame.Stats.TotalRoundsMapCsItaly}}      - {{.UserStatsForGame.Stats.TotalWinsMapCsItaly}}],
	['Militia',    {{.UserStatsForGame.Stats.TotalWinsMapCsMilitia}},    {{.UserStatsForGame.Stats.TotalRoundsMapCsMilitia}}    - {{.UserStatsForGame.Stats.TotalWinsMapCsMilitia}}],
	['Office',     {{.UserStatsForGame.Stats.TotalWinsMapCsOffice}},     {{.UserStatsForGame.Stats.TotalRoundsMapCsOffice}}     - {{.UserStatsForGame.Stats.TotalWinsMapCsOffice}}],
	['Aztec',      {{.UserStatsForGame.Stats.TotalWinsMapDeAztec}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeAztec}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeAztec}}],
	['Bank',       {{.UserStatsForGame.Stats.TotalWinsMapDeBank}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeBank}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeBank}}],
	['Cobblestone',      {{.UserStatsForGame.Stats.TotalWinsMapDeCbble}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeCbble}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeCbble}}],
	['Dust',       {{.UserStatsForGame.Stats.TotalWinsMapDeDust}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeDust}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeDust}}],
	['Dust2',      {{.UserStatsForGame.Stats.TotalWinsMapDeDust2}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeDust2}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeDust2}}],
	['Inferno',    {{.UserStatsForGame.Stats.TotalWinsMapDeInferno}},    {{.UserStatsForGame.Stats.TotalRoundsMapDeInferno}}    - {{.UserStatsForGame.Stats.TotalWinsMapDeInferno}}],
	['Lake',       {{.UserStatsForGame.Stats.TotalWinsMapDeLake}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeLake}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeLake}}],
	['Nuke',       {{.UserStatsForGame.Stats.TotalWinsMapDeNuke}},       {{.UserStatsForGame.Stats.TotalRoundsMapDeNuke}}       - {{.UserStatsForGame.Stats.TotalWinsMapDeNuke}}],
	['Safehouse',  {{.UserStatsForGame.Stats.TotalWinsMapDeSafehouse}},  {{.UserStatsForGame.Stats.TotalRoundsMapDeSafehouse}}  - {{.UserStatsForGame.Stats.TotalWinsMapDeSafehouse}}],
	['Shorttrain', {{.UserStatsForGame.Stats.TotalWinsMapDeShorttrain}}, {{.UserStatsForGame.Stats.TotalRoundsMapDeShorttrain}} - {{.UserStatsForGame.Stats.TotalWinsMapDeShorttrain}}],
	['Stmarc',     {{.UserStatsForGame.Stats.TotalWinsMapDeStmarc}},     {{.UserStatsForGame.Stats.TotalRoundsMapDeStmarc}}     - {{.UserStatsForGame.Stats.TotalWinsMapDeStmarc}}],
	['Sugarcane',  {{.UserStatsForGame.Stats.TotalWinsMapDeSugarcane}},  {{.UserStatsForGame.Stats.TotalRoundsMapDeSugarcane}}  - {{.UserStatsForGame.Stats.TotalWinsMapDeSugarcane}}],
	['Train',      {{.UserStatsForGame.Stats.TotalWinsMapDeTrain}},      {{.UserStatsForGame.Stats.TotalRoundsMapDeTrain}}      - {{.UserStatsForGame.Stats.TotalWinsMapDeTrain}}],
	['Vertigo',    {{.UserStatsForGame.Stats.TotalWinsMapDeVertigo}},    {{.UserStatsForGame.Stats.TotalRoundsMapDeVertigo}}    - {{.UserStatsForGame.Stats.TotalWinsMapDeVertigo}}]
```
Let's use the names for the maps as they are shown in game in the graph. It think it's shorter and looks nicer.